pdoc/doc_pyi.py

```diff
@@ -25,10 +26,17 @@ def find_stub_file(module_name: str) -> Path | None:
     """Try to find a .pyi file with type stubs for the given module name."""
     module_path = module_name.replace(".", "/")
 
-    for dir in sys.path:
+    # Find .pyi-file in a PEP 0561 compatible stub-package
+    module_part_name = module_name.split(".")
```

Comment:
This won\t work because we replace `.` with `/` a few lines above.

Reply:
We don't change `module_name`

```
>>> module_name = "example.submodule"
>>> module_path = module_name.replace(".", "/")
>>> module_name
example.module_name
>>> module_path
example/module_name
```

Reply:
Ah, didn't read properly. Thanks! 🍰